Sources/SwiftProtobuf/DoubleParser.swift

```diff
@@ -21,46 +21,39 @@ internal class DoubleParser {
     // In theory, JSON writers should be able to represent any IEEE Double
     // in at most 25 bytes, but many writers will emit more digits than
     // necessary, so we size this generously.
-    #if swift(>=4.1)
-      private var work =
-          UnsafeMutableRawBufferPointer.allocate(byteCount: 128,
-                                                 alignment: MemoryLayout<UInt8>.alignment)
-    #else
-      private var work = UnsafeMutableRawBufferPointer.allocate(count: 128)
-    #endif
+    private var work = 
+      UnsafeMutableBufferPointer<Int8>.allocate(capacity: 128)
```

Comment:
Just checking, no swift conditional needed now?

Reply:
That's correct, allocating typed pointers has never required an alignment specification as the alignment will naturally be `MemoryLayout<Element>.alignment`.